### ISSUE TITLE
Errors thrown in scripts are not shown and not failing the build.

### ISSUE THREAD

**Issue by DixonDs** [SharpeRAD/Cake.Powershell#24]
Error.ps1:
```
throw "My error";
```
build.cake:
```
#addin "Cake.Powershell"

var target = Argument("target", "Default");

Task("Default")
    .Does(() =>
{
	StartPowershellFile("error.ps1");
});

RunTarget(target);
```

When you run "error.ps1" directly, it outputs the error to the powershell console. But if you run standard "build.ps1" invoking "build.cake" above, it doesn't output any error and shows the build as successful.

**Comment by matkoch** [comment]
As far as I've noticed, the exit code is the first element contained in the returned collection.

**Comment by SharpeRAD** [comment]
This should probably be changed, it was introduced in #23. A more consistent approach would be to not output the exist code and throw errors.

Happy to accept a PR for the change as there are more than enough votes for it 👍

**Comment by yanigisawa** [comment]
I'm running into the same behavior, I'm taking a look at this issue.

**Comment by SharpeRAD** [comment]
Fixed in v0.3.5 courtesy of @yanigisawa 👍
